### ISSUE TITLE
Easier ability for users to set event notification filters

### ISSUE THREAD

**Issue by coolacid** [MISP/MISP#5900]
User settings as the ability to set "publish_alert_filter" this makes it so users are able to filter out event notifications based on tags. 

I'd propose moving this from a JSON blob that needs to be parsed to two selection boxes similar to the add tag to event. 

These boxes should be setup with `include` and `exclude` tags. An empty `include` box would default to all events. 

This would allow users a much simpler way to mange tag based event filters.

**Comment by iglocska** [comment]
Indeed, this definitely needs some love. It was kind of a proof of concept that was left as is, but having a decent interface to add filters would be great.
